simplesamlphp/config/config.php

```diff
@@ -1039,4 +1039,8 @@
      */
     'store.redis.prefix' => 'SimpleSAMLphp',
 );
-include 'acquia_config.php';
\ No newline at end of file
+include 'acquia_config.php';
```

Comment:
Do you want to only add the acquia config on the acquia environment?

Reply:
This line was created by the BLT command to initiate saml